### ISSUE TITLE
Winetricks fails to download W2KSP4_EN.exe

### ISSUE THREAD

**Issue by nolsen42** [roblox-linux-wrapper/roblox-linux-wrapper#176]
![image](https://user-images.githubusercontent.com/7462688/35362847-a9013b8a-012c-11e8-81c7-20c2b1c25947.png)

I almost got it to work with the rlw, and it let me move around for a bit before it kicked me off, so could this be a new issue? Perhaps a cheat detection or something?

**Comment by nolsen42** [comment]
Wine has gotten better, and now with the release of 3.0 (Including better support for DirectX 10 and 11), it might be worth giving another shot at trying to get it to launch.

**Comment by nolsen42** [comment]
Also, this may or may not help, but when reinstalling ROBLOX, I noticed some or one of the links you download from Microsoft's website is a 404.

```Downloading http://download.microsoft.com/download/E/6/A/E6A04295-D2A8-40D0-A0C5-241BFECD095E/W2KSP4_EN.EXE to /home/nolsen/.cache/winetricks/win2ksp4
--2018-01-24 17:45:20--  http://download.microsoft.com/download/E/6/A/E6A04295-D2A8-40D0-A0C5-241BFECD095E/W2KSP4_EN.EXE
Resolving download.microsoft.com (download.microsoft.com)... 23.208.128.203
Connecting to download.microsoft.com (download.microsoft.com)|23.208.128.203|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-01-24 17:45:21 ERROR 404: Not Found.

------------------------------------------------------
Downloading http://download.microsoft.com/download/E/6/A/E6A04295-D2A8-40D0-A0C5-241BFECD095E/W2KSP4_EN.EXE failed
------------------------------------------------------
```

**Comment by alfonsojon** [comment]
We need to either update our build of Winetricks or determine if ```wininet``` is even necessary with Wine 3.0. Thanks for the report of that issue. As for the "Kicked by server" issue, this is a new name for the "game has shut down" issue, which is already known.

**Comment by GLolol** [comment]
(RLW no longer uses a bundled Winetricks)

The W2KSP4_EN.exe issue is https://github.com/Winetricks/winetricks/issues/774, which is fixed in release 20170517

**Comment by alfonsojon** [comment]
@GLolol Thanks for the correction. I'm going to test it out to see if we even need to use the ```wininet``` verb as of Wine 3.0 so that maybe we can remove that verb as a whole.

**Comment by alfonsojon** [comment]
can confirm this is fixed as of wine-staging 5.4 and the current version of winetricks.
